From 941fa1f6e31bc7cb1b014277de36b80425da7010 Mon Sep 17 00:00:00 2001
From: Tony Jones <tonyj@suse.com>
Date: Wed, 5 Feb 2014 14:38:07 -0800
Subject: [PATCH 2/2] i386: fix redefinition error for e820entry

At least on our systems, xenctrl.h defines (unguarded) struct e820entry
Move the (guarded) definition in include/x86/x86-linux.h to below.

Signed-off-by: Tony Jones <tonyj@suse.de>
Signed-off-by: Simon Horman <horms@verge.net.au>
---
 kexec/arch/i386/crashdump-x86.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/kexec/arch/i386/crashdump-x86.c b/kexec/arch/i386/crashdump-x86.c
index 53bd077..979c2bd 100644
--- a/kexec/arch/i386/crashdump-x86.c
+++ b/kexec/arch/i386/crashdump-x86.c
@@ -41,12 +41,13 @@
 #include "../../crashdump.h"
 #include "kexec-x86.h"
 #include "crashdump-x86.h"
-#include "x86-linux-setup.h"
 
 #ifdef HAVE_LIBXENCTRL
 #include <xenctrl.h>
 #endif /* HAVE_LIBXENCTRL */
 
+#include "x86-linux-setup.h"
+
 #include <x86/x86-linux.h>
 
 extern struct arch_options_t arch_options;
-- 
1.8.5.3

